examples/server-rendering/server.js

```diff
@@ -0,0 +1,80 @@
+import express from 'express';
+import webpack from 'webpack';
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import {Provider} from 'react-redux';
+import {createStore} from 'redux';
+import {ReduxRouter} from '../../src'; // 'redux-router'
+import {reduxReactRouter, match} from '../../src/server'; // 'redux-router/server';
+import qs from 'query-string';
+import serialize from 'serialize-javascript';
+
+import webpackDevMiddleware from 'webpack-dev-middleware';
+import webpackHotMiddleware from 'webpack-hot-middleware';
+
+import config from './webpack.config.clientDev';
+import {MOUNT_ID} from './constants';
+import reducer from './reducer';
+import routes from './routes';
+
+const app = express();
+const compiler = webpack(config);
+
+const getMarkup = (store) => {
+  const initialState = serialize(store.getState());
+
+  const markup = renderToString(
+    <Provider store={store} key="provider">
+      <ReduxRouter/>
+    </Provider>
+  );
+
+  return `<!doctype html>
+    <html>
+      <head>
+        <title>Redux React Router – Server rendering Example</title>
+      </head>
+      <body>
+        <div id="${MOUNT_ID}">${markup}</div>
+        <script>window.__initialState = ${initialState};</script>
+        <script src="/static/bundle.js"></script>
+      </body>
+    </html>
+  `;
+};
+
+app.use(webpackDevMiddleware(compiler, {
+  noInfo: true,
+  publicPath: config.output.publicPath
+}));
+
+app.use(webpackHotMiddleware(compiler));
+
+app.use((req, res) => {
+  const store = reduxReactRouter({ routes })(createStore)(reducer);
```

Comment:
Maybe use `{ routes, createHistory: createMemoryHistory }` instead of `{ routes }`?